tensorflow_addons/register.py

```diff
@@ -0,0 +1,53 @@
+import glob
+import os
+from pathlib import Path
+
+import tensorflow as tf
+
+from tensorflow_addons.utils.resource_loader import get_project_root
+
+
+def register_all(keras_objects: bool = True, custom_kernels: bool = True) -> None:
```

Comment:
Maybe you should add docstring here to describe what it does and when it is required?